{{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/base/tests.py

```diff
@@ -5,4 +5,5 @@
 class HomeTests(TestCase):
 
     def test_base(self):
-        self.client.get(reverse('home'))
+        response = self.client.get(reverse('home'))
+        assert b'csrfmiddlewaretoken' in response.content
```

Comment:
Why a byte string? If there's a strong reason for sticking to simplicity, there should be a comment.